DTS-STN/Service-Canada-Design-System
# fix: remove border and related padding from Message.js

# DS-##Message.js

**Reminder: Add all changes for this PR to the CHANGELOG.md**

Main Reviewer: @zeddotes 

#### Description & Background Context:

Remove Message.js border and border padding

https://dev.azure.com/VP-BD/DECD/_boards/board/t/Design%20System/Stories%20and%20Activities/?workitem=86081

#### Where should the reviewer start?

File name: Message.js

#### How should this be tested?

View in storybook, see that there is no border on the Message component per the task linked

#### Are there any breaking changes?

None
